### PR TITLE
Verify packages for manually installed php on centos

### PR DESCRIPTION
**Pull request by pawelchcki**
### Readiness checklist
- [x] [Changelog entry](docs/changelog.md) added, if necessary
- [x] Tests added for this feature/bug

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -459,19 +459,19 @@ jobs:
...
   build_packages:
     jobs:
       - "5.6 20131106": &BUILD_PACKAGE_FORKFLOW
-          filters:
-            tags:
-              only: /(^build$)|(^[v]?[0-9]+(\.[0-9]+)*$)/
-            branches:
-              # Always build on master
-              ignore: /^(?!master$).*/
+          filters: {}
```
Comment (labbati) [inline .circleci/config.yml]: is this intended?

Comment (pawelchcki) [reply .circleci/config.yml]: No - its just temporary - to unblock the CI. I'll remove it before merging

File: package/post-install.sh [hunk 2]
```diff
@@ -8,15 +8,81 @@ INI_FILE_NAME='99-ddtrace.ini'
...
+    println
+    println "    extension=${EXTENSION_FILE_PATH}"
+    println
+    println "Note that your PHP API version must match the extension's API version"
+    println "PHP API version can be found using following command"
+    println
+    println "    php -i | grep 'PHP API'"
+    println
+
+    exit 0 # exit - but do not fail the installtion
```
Comment (labbati) [inline package/post-install.sh]: the whole meaning of this is that we still want to succeed so user can go and add it manually. is it my interpretation correct?

Comment (pawelchcki) [reply package/post-install.sh]: Yes - exactly. If someone has a very custom setup it still should be possible to install the package correctly.

**Pull request merged**